.travis.yml

```diff
@@ -1,36 +1,55 @@
 language: python
 
-sudo: false
-
-python:
-  - 3.5
-
-env:
-  - export FAIL_ON_EXTERNAL_DEPRECATION='False'
-  - export FAIL_ON_EXTERNAL_DEPRECATION='True'
-
 matrix:
+  include:
+  - python: 3.5
+    env: export FAIL_ON_EXTERNAL_DEPRECATION='False'
+  - python: 3.5
+    env: export FAIL_ON_EXTERNAL_DEPRECATION='True'
+  - os: osx
+    language: generic
+    python: 3.5
+    env: export FAIL_ON_EXTERNAL_DEPRECATION='False'
+  - os: osx
+    language: generic
+    python: 3.5
+    env: export FAIL_ON_EXTERNAL_DEPRECATION='True'
```

Comment:
This entry is not needed, as there is no OS dependence here (unless OSX has separate dependencies?).